### PR TITLE
Add workshops page.  Update citation section.

### PR DESCRIPTION
**Pull request by kabilar**
Fix https://github.com/datajoint/datajoint-elements/issues/93

### CODE REVIEW

File: docs/community/workshops.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# Workshops
+Find us at the following upcoming and past workshops!
+
+## DataJoint Office Hours
+
+The DataJoint open source team will be offering monthly Office Hours.  These are public workshops on Zoom where we will offer two half-hour sessions of free hands-on support to teams using DataJoint software.
```
Comment (CBroz1) [inline docs/community/workshops.md:6]: I suggest removing this section in favor of a link to the support doc, to reduce maintenance. If not, suggesting a change of tense

```suggestion
The DataJoint open source team will be offering monthly Office Hours.  These are public workshops on Zoom where we offer two half-hour sessions of free hands-on support to teams using DataJoint software.
```

File: docs/community/workshops.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
++ INCF Assembly - September 12-16, 2022
+
++ Society for Neuroscience - November 12-16, 2022
+
+## Past workshops
+
++ UCL Neuropixels Course - October 19, 2021
+
++ INCF Neuroinformatics Training Week - August 30 - September 2, 2021
+     + [Session 1 Recording](https://vimeo.com/586441656)
```
Comment (CBroz1) [inline docs/community/workshops.md:25]: When do we want to merge this? Before Monday? If after is ok, I suggest swapping for our youtube links to better equip us to monitor traffic

**Comment by kabilar** [comment]
Also updated citation section.

**Review by CBroz1 — APPROVED** [review]

File: docs/community/workshops.md (hunk 1 above)
Comment (kabilar) [reply docs/community/workshops.md]: Thanks @CBroz1.  I have removed this section.

**Comment by kabilar** [comment]
Thanks @CBroz1.  I have incorporated your suggestions.

**Review by CBroz1 — APPROVED** [review]

**Pull request merged**